spark/v3.4/spark/src/test/java/org/apache/iceberg/spark/TestSparkDistributionAndOrderingUtil.java

```diff
@@ -1742,7 +2516,7 @@ public void testRangePositionDeltaUpdatePartitionedTable() {
   //                       LOCALLY ORDERED BY _spec_id, _partition, _file, _pos, date, days(ts)
   // merge mode is HASH -> CLUSTER BY _spec_id, _partition, date, days(ts) +
   //                       LOCALLY ORDER BY _spec_id, _partition, _file, _pos, date, days(ts)
-  // merge mode is RANGE -> RANGE DISTRIBUTE BY _spec_id, _partition, _file, date, days(ts) +
```

Comment:
This may be debatable. We initially included `_file` to split the deletes if too many but AQE should be a better option we are sure we won't split deletes per partition unless needed.